templates/PlayFab_API.h.ejs

```diff
@@ -16,6 +16,16 @@ namespace PlayFab
     class PlayFab<%- api.name %>API
     {
     public:
+        /// <summary>
+        /// Runs the IHttpPlugin's Update function to check for responses to HTTP requests (could be a platform specific HttpPluginImpl or user defiend)
+        /// This has global consequences since all api's share the same httpPlugin 
+        /// Therefore, only 1 api needs to call Update() 
+        /// It Doesn't matter which API, all will do the exact same thing.
+        /// Please note any later Update call in the same tick will probably be a wasted call
+        /// You may call it in this manner: 
+        ///    <%- api.name %>.Update();
+        /// whenever your own game ticks.
+        /// </summary>
```

Comment:
You reference IHttpPlugin here, but in subsequent places you reference it in different ways so it is not entirely clear to me what is going on. 
    If I were to guess it might go like this.

    Calls the Update function on your implementation of the IHttpPlugin to check for responses to HTTP requests.
    
    In the asside you reference HttpPluginImpl then later you reference httpPlugin (and IHttpPlugin early) how do they all interact.
    
    This has global consequences since all api's share the same httpPlugin  
    Therefore, only 1 api needs to call Update().
    It Doesn't matter which API, all will do the exact same thing.
    
    Is this actually 
    All api's (maybe call out client, server, admin) share the same httpPlugin. 
    This means that you only need to call Update() on one API to retrieve the responses for all APIs.
        
    Additional cals to Update (on any API) during the same tick is unlikely retrieve addtional responses.
    
    Call Update when your game ticks as follows:
    
        <%- api.name %>.Update();